deployments/kubernetes/install/scripts/install-cni.sh

```diff
@@ -78,6 +78,8 @@ ${CNI_NETWORK_CONFIG}
 EOF
 fi
 
+KUBECFG_FILE_NM=${KUBECFG_FILE_NM:-ZZZ-istio-cni-kubeconfig}
```

Comment:
nit: Can we have a real number prefix, like `99-istio-cni-kubeconfig` or so?

Reply:
The numerical prefix will always be before the alphabetical prefixes and @jwendell was having problems on minikube where the file is named something that has an alphabetical prefix, e.g. something like "minikube_cni.conf".  I forget the exact name.  So, `ZZZ-` puts it at the lexicographical end for both cases.

Reply:
Got it, thanks!